src/slider/utils/handlers.ts

```diff
@@ -67,11 +67,11 @@ export function createHandlers({
     if (!validKeyCodes.includes(key)) return;
     e.preventDefault();
     if (
-      [validKeys.left, validKeys.right, validKeys.up, validKeys.down].includes(
+        ([validKeys.left, validKeys.right, validKeys.up, validKeys.down] as ElementType<typeof validKeyCodes>[]).includes(
         key
       )
     ) {
-      const direction = [validKeys.right, validKeys.up].includes(key) ? 1 : -1;
+      const direction = ([validKeys.right, validKeys.up] as ElementType<typeof validKeyCodes>[]).includes(key) ? 1 : -1;
```

Comment:
I'm not a big fan of casting. Thinking that validKeys should have a proper type already 🤔 . But it doesn't feel like something that is needed to be fixed on another level now 👍